src/selfplay/tournament.cc

```diff
@@ -65,6 +66,8 @@ void SelfPlayTournament::PopulateOptions(OptionsParser* options) {
       "multiplexing";
   options->Add<StringOption>(kNnBackendOptionsStr, "backend-opts");
   options->Add<BoolOption>(kVerboseThinkingStr, "verbose-thinking") = false;
+  options->Add<FloatOption>(kResignPlaythroughStr, 0, 100, "resign-playthrough")
```

Comment:
0.0f and 100.0f (one of compilers shows warning otherwise, don't remember which but probably MSVC)

Reply:
Done (though others nearby don't have the `f`)